Arm/joint.hpp

```diff
@@ -74,5 +74,11 @@ class Joint
   {
     return mpu.Read();
   }
+  void SetSpeed(units::angular_velocity::revolutions_per_minute_t speed)
+  {
+  speed_ = speed;
+  motor.SetSpeed(speed_);
+  }
```

Comment:
```suggestion
  void SetSpeed(int/double/float speed)
  {
  // Then cast speed variable into revolutions_per_minute_t...
  speed_ = speed;
  motor.SetSpeed(speed_);
  }
```

Reply:
This removes all the casting needed to be done within rover_arm_system.cpp. Much cleaner imo

Reply:
Same goes for SetPosition() function